example/components/ProviderExample.tsx

```diff
@@ -1,17 +1,20 @@
-import { CoinbaseWallet } from '@web3-react/coinbase-wallet'
+import type { CoinbaseWallet } from '@web3-react/coinbase-wallet'
```

Comment:
nit: just import, don't use `type`, because it's a maintenance burden to upkeep

Reply:
Personally, I prefer the explicit use of `type` where I am requiring a specific package for type definitions only. However, it is correct that TypeScript removes imports not utilized as values by default (https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues), so I guess this nit makes sense.